tests/p_podman/10_podman_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Author: Carlos Rodriguez-Fernandez <carlosrodrifernandez@gmail.com>
+
+t_Log "Running $0 - podman tests"
+
+tests_in_order=(
+    "podman version"
+    "podman info"
+    "podman run --rm quay.io/centos/centos:stream${centos_ver} bash -c 'echo HELLO' | grep -q -e 'HELLO'"
+    "podman system service -t 1"
+    "touch ${HOME}/test.txt && \
+     podman run --rm --privileged -v ${HOME}/test.txt:/test.txt quay.io/centos/centos:stream${centos_ver} bash -c 'echo HELLO > /test.txt' && \
+     grep -q -e 'HELLO' ${HOME}/test.txt && \
+     rm -f ${HOME}/test.txt"
+    "printf \"FROM quay.io/centos/centos:stream${centos_ver}\nCMD echo 'HELLO'\n\" > ${HOME}/Containerfile && \
```

Comment:
We still use t_functional to test CentOS Linux 7 updates, which doesn't have a corresponding CentOS Stream container. You probably want to conditionalize the podman test itself or pull a container that doesn't rely on a matching centos_ver

Reply:
Thank you @bstinsonmhk. I went ahead and limited the test to centos stream 8 or greater. I ran the tests in centos 7, and they don't fully work because of the version of podman anyway.

Reply:
I will test this and if everything works I'll approve the request this morning my time.